Pizzeria.Tests/BasketDataServiceTests.cs

```diff
@@ -222,14 +222,66 @@ public async Task User_changes_basketItem_quantity()
                 Name = firstCatalogItem.Name,
                 Price = firstCatalogItem.Price,
             };
+            var resultStorageQuantity = firstCatalogItem.StorageQuantity - dto.Quantity;
             var sut = new BasketDataService(_dbContext);
 
             // Act
             var updatedItemId = await sut.UpdateItemAsync(dto);
 
             // Assert
             var updatedCatalogItem = await _dbContext.BasketItems.FindAsync(updatedItemId);
-            updatedCatalogItem.Quantity.Should().Be(22);
+            updatedCatalogItem!.Quantity.Should().Be(22);
+            updatedCatalogItem.CatalogItem.StorageQuantity.Should().Be(resultStorageQuantity);
+        }
+
+        [Fact]
+        public async Task User_wants_more_quantity_than_storage_quantity_when_updating_basket_item()
```

Comment:
too long and not really correct
`user try to order more than  we have on storage`

Reply:
Renamed